evaluation.py

```diff
@@ -29,11 +31,9 @@ def _vstack_images(self, img_list, interpolation=cv2.INTER_CUBIC):
         return cv2.vconcat(im_list_resize) 
 
     # Returns np array of predicted heatmaps for a given image and model
-    def predict_heatmaps(self, h, X):
-        h._load_model(self.model_json, self.weights)
-
+    def predict_heatmaps(self, X):
         X = np.expand_dims(X, axis=0) # add "batch" dimension of 1 because predict needs shape (1, 256, 256, 3)
-        predict_heatmaps = h.model.predict(X)
+        predict_heatmaps = self.model.predict(X)
         predict_heatmaps = np.array(predict_heatmaps) # output shape is (num_hg_blocks, 1, 64, 64, 17)
         return predict_heatmaps
```

Comment:
Moved the model loading to instance evaluation. On my laptop loading a model from disk takes ~10 seconds.